exercises/concept/vec-macro/.docs/instructions.md

```diff
@@ -0,0 +1,16 @@
+You are going to initialize empty buffers and list the first five numbers of the Fibonacci sequence.
```

Comment:
Maybe add a little introduction to the Fibonacci sequence here? It is unlikely that people haven't heard from it, but a reminder would be useful I feel.

Reply:
Agreed. I had it in originally but removed it in my zealous pruning!